app/src/main/java/org/mozilla/fenix/ext/String.kt

```diff
@@ -144,3 +145,31 @@ fun String.toEditable(): Editable = Editable.Factory.getInstance().newEditable(t
  */
 fun String?.replaceConsecutiveZeros(): String? =
     this?.replaceFirst(":0", ":")?.replace(":0", "")
+
+/**
+ * Returns true if the url contains any query parameters specified by the [searchParameters].
+ *
+ * @param searchParameters [String] of the following forms:
+ * - "" (empty) - Don't search for any params
+ * - "key" - Search param named "key" with any or no value
+ * - "key=" - Search param named "key" with no value
+ * - "key=value" - Search param named "key" with value "value"
+ */
+fun String.urlContainsQueryParameters(searchParameters: String): Boolean {
```

Comment:
This method looks to be highly specialized and usable in one specific scenario.
Can we avoid adding it to the String namespace? Maybe it can be added as an extension function to an `Uri` that is less used and already seen below in which case the "contains**QueryParameters**" name would also seem appropriate?